dev_tools/locustfile.py

```diff
@@ -0,0 +1,40 @@
+"""Locust (https://docs.locust.io/en/stable/what-is-locust.html) is a load testing tool.
+To run Locust, set the SUPERSTAQ_API_KEY environment variable and execute
+"locust --config=locust.conf" in the "dev_tools" directory. Then
+navigate to http://0.0.0.0:8089/, insert the number of users and spawn rate,
+then click the "Start swarming" button.
+"""
+
+from __future__ import annotations
+
+import logging
+from typing import TYPE_CHECKING
+
+import cirq_superstaq
+import requests
+from locust import HttpUser, events, task
+
+if TYPE_CHECKING:
+    from locust.env import Environment
```

Comment:
nit: why not just import
```suggestion
import locust.env
```
and use `locust.HttpUser` / `locust.events` / `locust.task` / `locust.env.Environment` below?

Reply:
Done